src/streamlink/stream/hls_playlist.py

```diff
@@ -102,22 +105,22 @@ def parse_attributes(self, value):
         def map_attribute(key, value, quoted):
             return (key, quoted or value)
 
-        attr = re.findall(ATTRIBUTE_REGEX, value)
+        attr = M3U8Parser._attr_re.findall(value)
```

Comment:
shouldn't this be `self.`

Reply:
I think both are possible

Reply:
Both are correct, it's an issue if you want to subclass and change the regex - which we might want to do in the future for LHLS (or similar).